components/CookieBanner.vue

```diff
@@ -21,11 +21,15 @@
 </template>
 
 <script setup>
+import { useGtm } from '@gtm-support/vue-gtm'
+
 const showBanner = useCookie('bg.showbanner', { default: () => true })
 const allowCookies = useCookie('bg.allowcookies', { default: () => false })
 function yesCookies() {
     showBanner.value = false
     allowCookies.value = true
+    const gtm = useGtm()
+    gtm.enable()
```

Comment:
We already check for `allowCookies` and enable GTM in the specific places where we use it (`composables/useContactForm.js` and `pages/index.vue`) is there a need to enable it here and in `plugins/vue-gtm.client.js`? Or we can enable GTM in a central place? I'm not too sure what's the best thing to do here though.

Btw should this be `gtm.enable(true)`? `gtm.enable()` only checks if gmt is enabled or not.

Reply:
`gtm.enable()` is defined with `true` as default, so it doesn't make a difference. Making it explicit is a good idea though, so I've added it all.

If a user has visited our site before and accepted cookies, we want to enable gtm when they load the page. For this, we need to enable it in the `vue-gtm` plugin. But I hear you on this whole thing being a bit messy. To me it's an acceptable mess because it's limited and I have this loose sense of reworking our analytics (i.e. removing all the unused bits) anyway. If you have a good idea to make it less messy, I'd be interested in hearing it though.

Reply:
You are right. I misread the document and confused `gtm.enable()` with `gtm.enabled()`.

I'm ok with erring on the side of redundancy to make sure this works so this looks good to me. It being messy is not really a problem now anw as you said coz of the small code size.